### PR TITLE
 Fix Missing Labels in FER2013 Dataset #8118

### PR DESCRIPTION
**Pull request by NSalberg**
In the current implementation,  labels are not provided for test split. 

The [FER2013 ](https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge)dataset provides labels for both training and testing in the icml_face_data.csv file. 

This pull request reads the data from the icml_face_data.csv instead, providing labels for both the train and test splits.

### CODE REVIEW

File: torchvision/datasets/fer2013.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ class FER2013(VisionDataset):
     """
 
     _RESOURCES = {
-        "train": ("train.csv", "3f0dfb3d3fd99c811a1299cb947e3131"),
-        "test": ("test.csv", "b02c2298636a634e8c2faabbf3ea9a23"),
+        "train": ("icml_face_data.csv", "b114b9e04e6949e5fe8b6a98b3892b1d"),
+        "test": ("icml_face_data.csv", "b114b9e04e6949e5fe8b6a98b3892b1d"),
```
Comment (NicolasHug) [inline torchvision/datasets/fer2013.py:27]: Thank you for the PR @NSalberg .

The feature request looks reasonable, however I think we should avoid forcing the existence of `icml_face_data.csv` because that will potentially break existing users who only have downloaded `train.csv` and `test.csv`.

Perhaps we should instead check the existence of `icml_face_data.csv`  and use it if it exists, and otherwise fall back to `train.csv` and `test.csv`?

**Comment by NicolasHug** [comment]
Thanks for the PR @NSalberg , I followed-up and made the requested changes in https://github.com/pytorch/vision/pull/8452
